zapd/models.py

```diff
@@ -511,6 +511,7 @@ def decline_view(self):
         # process the proposal
         if proposal.status == proposal.STATE_CREATED:
             proposal.status = proposal.STATE_DECLINED
+            proposal.authorizer = current_user
```

Comment:
this seems unrelated?

Reply:
Was originally trying to include the username in the authorized by when declined as well. Old stuff.... deleting it.